requirements/requirements.txt

```diff
@@ -10,5 +15,7 @@ tensorboard
 text-unidecode
 torch
 tqdm>=4.41.0
+transformer-engine @ git+https://github.com/NVIDIA/TransformerEngine.git@${TE_TAG}
```

Comment:
Remove TE from core. No compiled dependencies can added at all to pip requirements.txt file, especially core requirements